do_cmake.sh

```diff
@@ -24,17 +24,20 @@ if [ -r /etc/os-release ]; then
           if [ "$MAJOR_VER" -ge "8" ] ; then
               PYBUILD="3.6"
               ARGS+=" -DWITH_RADOSGW_AMQP_ENDPOINT=OFF"
+              ARGS+=" -DWITH_RADOSGW_KAFKA_ENDPOINT=OFF"
```

Comment:
why turn these OFF? can we not add the amqp and kafka dependencies to install-deps.sh?

Reply:
> why turn these OFF? can we not add the amqp and kafka dependencies to install-deps.sh?

i just copied what we did for amqp. not sure why it is turned off, as the install-deps script should take care of RHEL/Centos

Reply:
see commit message of a118ba0edadf5db8ccd171920982976ef75c9a75

Reply:
seems like both the rabbitmq and the rdkafka libs belong to the "power tools" repo in centos8.
can we assume this repo is available?

Reply:
@yuvalif yes! we do enable PowerTools repo in `install-deps.sh`. could you help review #30974 ?